### ISSUE TITLE
"bundle config --local path 'vendor/bundle'" takes no action

### ISSUE THREAD

**Issue by kevinburkemeter** [rubygems/rubygems#3385]
As of version 2.0 running `bundle install --path vendor/bundle` suggests that the `--path` variable is deprecated.

The [documentation for bundler v2.0 suggests](https://bundler.io/man/bundle-config.1.html):

> Executing bundle config --local <name> <value> will set that configuration to the local application. The configuration will be stored in app/.bundle/config.

At the top level of my Rails application ("myserver"), I run `bundle config --local path 'vendor/bundle'`. 

I expect that this will install contents to `myserver/app/.bundle/config`, but instead, nothing happens.

```
$ ls -al app
total 0
drwxr-xr-x 10 kevin staff  320 Mar  4 14:36 .
drwxr-xr-x 32 kevin staff 1024 Mar 14 19:53 ..
drwxr-xr-x  7 kevin staff  224 Mar  4 14:36 assets
drwxr-xr-x  3 kevin staff   96 Mar  4 14:36 channels
drwxr-xr-x 11 kevin staff  352 Mar  4 16:33 controllers
drwxr-xr-x  5 kevin staff  160 Mar  4 14:36 helpers
drwxr-xr-x  3 kevin staff   96 Mar  4 14:36 jobs
drwxr-xr-x  3 kevin staff   96 Mar  4 14:36 mailers
drwxr-xr-x  6 kevin staff  192 Mar  4 15:54 models
drwxr-xr-x  7 kevin staff  224 Mar  4 14:36 views
```

**Comment by kevinburkemeter** [comment]
I do see something at the top level, in `myserver/.bundle` which raises the possibility that `app/.bundle` is supposed to be a case where `app` is the folder containing e.g. `.git`, `Gemfile`, etc.

However, that's pretty confusing when Rails also has a top level folder named `app` that includes controllers, views etc. Perhaps a different name could be used to indicate the enclosing folder.

**Comment by colby-swandale** [comment]
The `path` configuration needs to be set in your project root directory where the `Gemfile` lives.

**Comment by deivid-rodriguez** [comment]
Yes, the documentation is very confusing as @kevinburkemeter points out, specially since `app` is a standard folder on every Rails application.

I believe the documentation should read as

> Executing `bundle config --local` will set that configuration to the local application. The configuration will be stored in `./.bundle/config`.

rather than

> Executing `bundle config --local` will set that configuration to the local application. The configuration will be stored in `app/.bundle/config`.

I would accept such a PR for the doc site.

**Comment by kevinburkemeter** [comment]
I tried installing GNU `col` instead of the BSD one, also no luck, a bunch of weird escapes show up in the output. OK. Let me just submit the change, I guess.

**Comment by deivid-rodriguez** [comment]
I'm trying to fix this in #3391, let's see if it works.
